src/features/paste/actions/paste.actions.ts

```
@@ -0,0 +1,1136 @@
+    }
+
+    // Build order by
+    let orderBy;
+    switch (sort) {
+      case "oldest":
+        orderBy = paste.createdAt;
+        break;
+      case "views":
+        orderBy = desc(paste.views);
+        break;
+      case "newest":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)